recipes/enzy_htp/license.md

```diff
@@ -0,0 +1 @@
+© 2021-2024, Vanderbilt University, All Rights Reserved
```

Comment:
This license is too short. There is no information about redistribution. Is this really the correct one?

Reply:
Thank you for the comment!
Yes, I'm working with the tech transfer office of Vanderbilt University to finalize the formal license for the package but it takes insanely long to finish. Is it possible to have this license here at the beginning and I will update the formal license when it is done. 
BTW, is the license file here supposed to be for the recipe or the package? If it is for the recipe, I can change it to a general one or just delete it.